src/components/Dropdown/Dropdown.tsx

```diff
@@ -96,9 +101,43 @@ const OptionsContainer = styled(Div)`
           `
         : ''
     }
-    border-top: 0px solid transparent;
-    border-radius: 0rem 0rem 0.25rem 0.25rem;
     z-index: 1000;
+    ${
+      isOpenedBelow
+        ? `
+          top: 100%;
+          border-top: 0px solid transparent;
+          border-radius: 0rem 0rem 0.25rem 0.25rem;
+        `
+        : `
+          top: auto;
+          bottom: 100%;
+          border-bottom: 0px solid transparent;
+          border-radius: 0.25rem 0.25rem 0rem 0rem;
+        `
```

Comment:
might be worth trying to do a similar thing as above where we conditionally switch between "bottom" and "top" here